java/src/jmri/jmrit/catalog/PreviewDialog.java

```diff
@@ -446,6 +432,8 @@ private long availableMemory() {
             for (int i = 0; i < memoryTest.size(); i++) {
                 memoryTest.remove(i);
             }
+            System.gc();
```

Comment:
I added it in hopes of improving performance, but wasn't really convinced it helped.  I'll remove it.